bundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
We don't need this class, as it is empty. Feel free to remove it.

Reply:
fixed in f94a59bf2f9a236ac8c665c4c0cd416db5b45858